test/models/programlocation.js

```diff
@@ -0,0 +1,33 @@
+const assert = require('assert');
+const _ = require('lodash');
+const fs = require('fs-extra');
+const path = require('path');
+const { v4: uuid } = require('uuid');
+
+const helper = require('../helper');
+const models = require('../../models');
+
+describe('models.programlocation', () => {
+  beforeEach(async () => {
+    await helper.loadFixtures(['programlocation']);
+  });
+
+  it('create a new program location record', async () => {
+    let ProgramLocation = models.ProgramLocation.build({
+      LocationId: 1,
+      ProgramId: 1,
+    });
+    assert.deepStrictEqual(ProgramLocation.id, null);
+    await ProgramLocation.save(); //save to data base, id will generate after it save
+    assert(ProgramLocation.id);
+
+    assert.deepStrictEqual(ProgramLocation.LocationId, 1);
+    assert.deepStrictEqual(ProgramLocation.ProgramId, 2);
```

Comment:
This ProgramId value of 2 is not what you specified above on line 18, which is 1, which is causing this to fail.